Conch/bin/conch-create-user

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env perl
+
+use v5.20;
+use warnings;
+
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+
+use Conch;
+use Conch::Model::User;
+use Getopt::Long::Descriptive;
+use Conch::Mail;
+
+my ($opt, $usage) = describe_options(
+    'conch-create-user %o',
+    [ 'name|n=s',       'the user\'s name' ],
+    [ 'email|e=s',      'the user\'s email address', { required => 1 } ],
+    [ 'password|p=s',   'the user password', { default => random_password() } ],
+    [ 'send-email',     'send email to user' ],
+    [],
+    [ 'help',           'print usage message and exit', { shortcircuit => 1 } ],
+);
+
+print($usage->text), exit if $opt->help;
+
+my $conch = Mojo::Server->new->build_app('Conch');
+my $schema = Conch::DB::Schema->connect($conch->config('pg'));
```

Comment:
if you're loading up the Conch mojo app, there's a 'schema' helper available that does the needful. https://github.com/joyent/conch/blob/master/Conch/lib/Conch.pm#L54

Unless I'm misunderstanding which is always possible :)

Reply:
that's a Mojo schema object, relies on the Mojo way of connecting, which is the bits I'm aiming to remove. That's why I added new connect logic right in Conch::DB::Schema (likely soon to be renamed Conch::DB, although it `isa->('DBIx::Class::Schema')`).

Reply:
The reason why I'm loading up the app there is to get at the config tree.. there's probably a better/faster way of doing that, so I'll replace that when I find it.

Reply:
that helper is returning a Conch::DB::Schema object configured using the data in the mojo config.